### DISCUSSION TITLE
Turns on the lights of Spike Prime Color Sensor at the specified brightness.

### DISCUSSION THREAD

**Discussion by MohammedEsmaeel** [pybricks/support#1703] [Q&A]
Hello,

I have tried this simple code to change the lights of the Color Sensor,

```python
from pybricks.pupdevices import ColorSensor 
from pybricks.parameters import Port 
from pybricks.tools import wait 
# Initialize the sensor. 
sensor = ColorSensor(Port.A)

# Turn on one light at a time, at half the brightness. 

sensor.lights.on([50, 50, 50])


color = sensor.color()

# Print the color. 
print(color)
```

When Running the line `sensor.lights.on([50, 50, 50])` the color sensor successfully lights on half the brightness, But when the next line run `color = sensor.color()` the color sensor back to it's normal light (full light).

My goal is to change the brightness and read color on this specific brightness.

**Comment by laurensvalk** [comment]
Color scanning always happens at full brightness, so it switches back automatically.

If you mean to calibrate your colors, you can do so with the [`detectable_colors` method](https://docs.pybricks.com/en/latest/pupdevices/colorsensor.html#changing-the-detectable-colors).
